src/DataTable/DataTable.jsx

```diff
@@ -67,22 +74,20 @@ export class DataTable extends Component {
     window.removeEventListener('resize', this.handleResize)
     grid.api.destroy()
   }
-  
-  exportToCSV() {
+
+  exportToCSV = () => {
```

Comment:
This doesn't need need to have an anonymous function asigned to it. It can just be a class method.